tests/unit/test_session.py

```diff
@@ -1,195 +1,207 @@
+
+
+def test___session___open_with_context_manager_when_init_fails___raises_driver_error(lib_mock):
+    def init_error_mock(resource_name, id_query, reset_device, vi):
+        vi.contents.value = SESSION_HANDLE.value
+        return nisync.errors.Status.DEVICE_NOT_FOUND
+
+    lib_mock.niSync_init.side_effect = init_error_mock
+
+    with pytest.raises(nisync.errors.DriverError) as exc:
+        with nisync.Session(RESOURCE_NAME):
+            ...
+
+    assert exc.value.code == nisync.errors.Status.DEVICE_NOT_FOUND
+
+
+def test___session___open_with_reset___succeeds(lib_mock):
+    with nisync.Session(RESOURCE_NAME, reset_device=True):
+        ...
+
+    expected_calls = [
+        mock.call.niSync_init(RESOURCE_NAME.encode(), VI_FALSE, VI_TRUE, mock.ANY),
+        mock.call.niSync_close(SESSION_HANDLE),
+    ]
+    assert lib_mock.mock_calls == expected_calls
+
+
+def test___session___reset___succeeds(lib_mock):
+    lib_mock.niSync_reset.return_value = 0
+
+    with nisync.Session(RESOURCE_NAME) as session:
+        session.reset()
+
+    expected_calls = [
+        mock.call.niSync_init(RESOURCE_NAME.encode(), VI_FALSE, VI_FALSE, mock.ANY),
+        mock.call.niSync_reset(SESSION_HANDLE),
+        mock.call.niSync_close(SESSION_HANDLE),
+    ]
+    assert lib_mock.mock_calls == expected_calls
+
+
+def test___session___self_test___succeeds(lib_mock):
+    def self_test_mock(vi, test_result, test_message):
+        test_result.value = 0
+        test_message.value = b""
+        return 0
+
+    lib_mock.niSync_self_test.side_effect = self_test_mock
+
+    with nisync.Session(RESOURCE_NAME) as session:
+        result, message = session.self_test()
+
+    assert result == 0
+    assert message == ""
+    expected_calls = [
+        mock.call.niSync_init(RESOURCE_NAME.encode(), VI_FALSE, VI_FALSE, mock.ANY),
+        mock.call.niSync_self_test(SESSION_HANDLE, mock.ANY, mock.ANY),
+        mock.call.niSync_close(SESSION_HANDLE),
+    ]
+    assert lib_mock.mock_calls == expected_calls
+
+
+def test___session___connect_clock_terminals___succeeds(lib_mock):
+    lib_mock.niSync_ConnectClkTerminals.return_value = 0
+
+    with nisync.Session(RESOURCE_NAME) as session:
+        session.connect_clock_terminals(source=OSCILLATOR, destination=PXI_CLK10_IN)
+
+    expected_calls = [
+        mock.call.niSync_init(RESOURCE_NAME.encode(), VI_FALSE, VI_FALSE, mock.ANY),
+        mock.call.niSync_ConnectClkTerminals(
+            SESSION_HANDLE, OSCILLATOR.encode(), PXI_CLK10_IN.encode()
+        ),
+        mock.call.niSync_close(SESSION_HANDLE),
+    ]
+    assert lib_mock.mock_calls == expected_calls
+
+
+def test___session___disconnect_clock_terminals___succeeds(lib_mock):
+    lib_mock.niSync_DisconnectClkTerminals.return_value = 0
+
+    with nisync.Session(RESOURCE_NAME) as session:
+        session.disconnect_clock_terminals(source=OSCILLATOR, destination=PXI_CLK10_IN)
+
+    expected_calls = [

```

Comment:
Do you know why every line in this file shows as modified? Did the line endings change or something?

Reply:
I ran command "poetry run ni-python-styleguide lint" and "poetry run ni-python-styleguide fix" to fulfill the requirement in CONTRIBUTING.md.  

The [last change](https://github.com/ni/nisync-python/pull/18/commits/693777cbdbf96b08fca9c0d1ad800a8f1f674f04) in this PR is the results of running "poetry run ni-python-styleguide fix".